src/EditorFeatures/CSharpTest/AddStaticModifier/AddStaticModifierTests.cs

```diff
@@ -0,0 +1,111 @@
+﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+
+using Microsoft.CodeAnalysis.CSharp.AddStaticModifier;
+using System.Threading.Tasks;
+using Microsoft.CodeAnalysis.CodeFixes;
+using Microsoft.CodeAnalysis.Diagnostics;
+using Microsoft.CodeAnalysis.Editor.CSharp.UnitTests.Diagnostics;
+using Microsoft.CodeAnalysis.Test.Utilities;
+using Xunit;
+
+namespace Microsoft.CodeAnalysis.Editor.CSharp.UnitTests.AddStaticModifier
+{
+    public class AddStaticModifierTests : AbstractCSharpDiagnosticProviderBasedUserDiagnosticTest
```

Comment:
How extensive shall I be with the tests? For example, I haven't added any test with other modifier or explicit accessibility.

Reply:
I think current set of tests looks reasonable to me.